test/scan.h

```diff
@@ -159,7 +159,7 @@ struct scan_handler : error_handler {
   const char* pos() const { return scan_ctx_.begin(); }
 
   void on_text(const char* begin, const char* end) {
-    auto size = end - begin;
+    auto size = static_cast<size_t>(end - begin);
```

Comment:
Please use `to_unsigned` here and below.

Reply:
OK